modules/creating-serverless-apps-yaml.adoc

```diff
@@ -1,22 +1,23 @@
 // Module included in the following assemblies:
 //
-// * serverless/getting-started-knative-services.adoc
+// serverless/serving-creating-managing-apps.adoc
 
-[id="deploying-serverless-apps_{context}"]
-= Deploying a serverless application
+[id="creating-serverless-apps-yaml_{context}"]
+= Creating serverless applications using YAML
 
-To deploy a serverless application, you must apply the `service.yaml` file.
+To create a serverless application, you can create a YAML file and apply it using `oc apply`.
+
+In this example, the YAML file is named `service.yaml`. You must substitute the name of your YAML file for these commands if the file has a different name.
```

Comment:
kind of feels like too much hand-holding here? (maybe just the first sentence would be enough to warn the user that the "service.yaml" is an example file name used in the following example, I don't know... )

Reply:
Yeah, I'd agree, but I think this was called out in a previous OCP doc review, i.e. someone asked if the yaml *must* be named that :woman_shrugging:

Reply:
How about we rename it to hello-service.yaml , and keep just the first sentence "In this example, the YAML file is named `hello-service.yaml` ?